lib/templates/haml/scaffold/_form.html.haml

```diff
@@ -1,10 +0,0 @@
-= simple_form_for(@<%= singular_table_name %>) do |f|
```

Comment:
Looks like it's from default Rails generator. We don't use it anywhere. That's why I deleted it.